doc/modules/model_evaluation.rst

```diff
@@ -1378,6 +1379,36 @@ function::
     for an example of mean squared error usage to
     evaluate gradient boosting regression.
 
+.. _mean_squared_log_error:
+
+Mean squared log error
+----------------------
+
+The :func:`mean_squared_log_error` function computes a risk metric corresponding
+to the expected value of the logarithmic squared (quadratic) error or loss.
```

Comment:
Oops, thanks for pointing this out. I would have missed it completely ! Changing it.